gspread/worksheet.py

```diff
@@ -464,7 +503,13 @@ def get_values(self, range_name=None, combine_merged_cells=False, **kwargs):
             worksheet.get_values('A2:B4', value_render_option=ValueRenderOption.formula)
         """
         try:
-            vals = fill_gaps(self.get(range_name, **kwargs))
+            if maintain_size:
```

Comment:
should be the other way around right ? if `maintain_size` is `True` we call the get range size, then fill gaps from the computed range size and not just the values right ?